client/components/empty-content/package.json

```diff
@@ -1,6 +0,0 @@
-{
```

Comment:
Peculiar eh? In the early days of Calypso, each component directory had its own `package.json` and the main file was often named after the component (rather than index.jsx).